workflow-runtime-android/src/main/java/com/squareup/workflow1/android/AndroidRenderWorkflow.kt

```diff
@@ -141,10 +140,12 @@ public fun <OutputT, RenderingT> renderWorkflowIn(
  * in any workflows, after the initial render pass, will be handled by this scope, and cancelling
  * this scope will cancel the workflow runtime and any running workers. Note that any dispatcher
  * in this scope will _not_ be used to execute the very first render pass (which happens
- * synchronously).
- * Also note that if there is no [CoroutineDispatcher] in this scope then Compose UI's
- * [AndroidUiDispatcher.Main] will be used as this is the most performant Android UI dispatcher
- * for Workflow.
+ * synchronously). We recommend using a `CoroutineDispatcher` in this scope's `CoroutineContext`
+ * that will ensure that all dispatched coroutines are run before the next Choreographer frame.
+ * Compose UI's `AndroidUiDispatcher.Main` provides this behavior in a performant way!
```

Comment:
Should update all the compose samples to demonstrate.

Reply:
* Could also mention that compose apps can use `Workflow.renderAsState` to sidestep thinking about this. Too much?
* Did we ever extend `renderAsState` to allow a config to be passed in?

Reply:
That's a good point about the samples!

Reply:
> Did we ever extend renderAsState to allow a config to be passed in?

I don't think so!

Reply:
> Did we ever extend renderAsState to allow a config to be passed in?


Turns out we did.